mappings/net/minecraft/block/Block.mapping

```diff
@@ -288,7 +376,16 @@ CLASS net/minecraft/class_17 net/minecraft/block/Block
 	METHOD method_1627 getTexture (II)I
 		ARG 1 side
 		ARG 2 meta
-	METHOD method_1630 (Lnet/minecraft/class_18;III)V
+	METHOD method_1628 onBreak (Lnet/minecraft/class_18;Lnet/minecraft/class_54;IIII)V
```

Comment:
Should instead be `afterBreak`, because in yarn it's the method that increases breaking stat